specification/cognitiveservices/OpenAI.Inference/models/translation.create.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+import "./transcription.create.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Translation request.")
+model TranslationOptions {
+
+    @doc("The audio file to translate. Binary data.")
+    // how do we handle binary format for a member?
+    file: string;
+
+    @doc("An optional text to guide the model's style or continue a previous audio segment. The prompt should be in English.")
+    prompt?: string;
+
+    @doc("The format of the transcription output, in one of these options: json, text, srt, verbose_json, or vtt.")
+    @projectedName("json", "response_format")
+    responseFormat?: TranscriptionFormat = TranscriptionFormat.json;
+
+    @doc("The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic. If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.")
+    temperature?: float32 = 0;
+}
+
+@doc("Translation response.")
+model Translation {
```

Comment:
Is there any practical difference between the transcription and translation response models? We may be able to converge these (potentially even the routes, though I'm not sure that's a good idea).

Reply:
`Transcription` has an additional field called `language` used as a support value to increase accuracy of the transacription. [Link](https://github.com/Azure/azure-rest-api-specs-pr/pull/14569/files#diff-a6feb0a5a2dd571ac74fb27e7d8a5d88040b02e6a67bd5fa989e41a87aee3616R1566)

Reply:
Sorry for the confusion. I realise your comment was on the response object, which yes, it was merged and I reflected that in the TSP now :) Thanks!